.dprint.jsonc

```diff
@@ -0,0 +1,32 @@
+{
+  "exec": {
+    "commands": [{
+      "command": "rustfmt",
+      "exts": ["nr"],
+    }],
+  },
+  "prettier": {
+    "singleQuote": true,
+  },
+  "typescript": {
+    "quoteProps": "asNeeded",
+    "quoteStyle": "preferSingle",
+    "semiColons": "asi",
+    "useBraces": "preferNone",
+  },
+  "excludes": [
+    "**/node_modules",
+    "yarn.lock",
+    ".yarn",
+    "packages/merkle-trees/src/globals.nr",
```

Comment:
As we are using `rustfmt`, we check against rust syntax, not nargo syntax.
Althought very similar, they are some differences. The only difference in this repo is [`global`](https://noir-lang.org/docs/noir/concepts/globals/#globals), so I am ignoring it.
As we can ignore only complete files, I had to move the `TREE_DEPTH` declaration in a dedicated file/module.

Reply:
I think it could be worth investigating why doesn't nargo fmt work in the first place. I'll reach out to the noir team